RNPurchases.podspec

```diff
@@ -14,14 +14,15 @@ Pod::Spec.new do |spec|
 
   spec.source       = { :git => "https://github.com/RevenueCat/react-native-purchases.git" }
   spec.source_files = "ios/**/*.{h,m,swift}"
+  spec.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
 
   # Ignore the downloaded Purchases.framework
   spec.exclude_files = [
     "ios/Purchases.framework",
     "ios/PurchasesHybridCommon.framework"
   ]
 
-  spec.dependency   "React"
-  spec.dependency   "PurchasesHybridCommon", '1.4.5'
+  spec.dependency   "React-Core"
```

Comment:
Did this change?

Reply:
yeah, this is from #191. It's what's breaking the tests, but we really should try to get it fixed before merging. Turns out that we never should have had "React" as the dependency, but it just happened to work on Xcode 11. It breaks with Xcode 12, though. https://github.com/facebook/react-native/issues/29633#issuecomment-694187116